packages/ui/src/components/editor/properties/fog/index.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Color } from 'three'
+
+import { useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
+import { FogSettingsComponent, FogType } from '@etherealengine/spatial/src/renderer/components/FogSettingsComponent'
+
+import {
+  EditorComponentType,
+  commitProperty,
+  updateProperty
+} from '@etherealengine/editor/src/components/properties/Util'
+import { GiFog } from 'react-icons/gi'
+import ColorInput from '../../../../primitives/tailwind/Color'
```

Comment:
I notice `<ColorInput />` looks different from figma design.


<img width="332" alt="Screenshot 2024-06-13 at 11 12 32 AM" src="https://github.com/EtherealEngine/etherealengine/assets/44120813/095c7566-ab80-43d1-a1dd-6f4247583a19">
<img width="332" alt="Screenshot 2024-06-13 at 11 12 32 AM" src="https://github.com/EtherealEngine/etherealengine/assets/44120813/095c7566-ab80-43d1-a1dd-6f4247583a19">

Reply:
there is slight color difference, it should be fine, i would like to find a means to style the generated color picker as well though